### DISCUSSION TITLE
postgres-backup-s3 : enable gzip

### DISCUSSION THREAD

**Discussion by nicolasmonfort** [eeshugerman/postgres-backup-s3#37] [Ideas]
Hello guys,

Could you add an install **gzip** into : https://github.com/eeshugerman/postgres-backup-s3/blob/master/src/install.sh
`apk install gzip`

in order to create dump compressed with this option:
PGDUMP_EXTRA_OPTS="| gzip -9"

Thank for your feedback

**Comment by eeshugerman** [comment]
There's no need for gzip because we use `pg-dump`'s `custom` format which is already compressed. See https://serverfault.com/a/440924

**Comment by nicolasmonfort** [comment]
Thanks Elliott for your feedback.

In deed, in the documentation postgresql (https://www.postgresql.org/docs/current/app-pgdump.html)
with "-F c" or "--format custom". It's a custom-fromat archive for input pg_restore. This format is also compressed by default.

But when we use gzip on this kind of custom file, we can win almost 25%. It could be insteresing on a huge file.

Could you please reconsider my proposal ?

**Reply by eeshugerman** [reply]
How did you arrive at the 25% number? What  size database did you test on?

This has come up in the past and we determined the difference becomes negligible (a few megabytes) for even fairly small databases (a few gigabytes): https://github.com/eeshugerman/postgres-backup-s3/issues/7

Btw sorry I meant to link that issue before but couldn't find it at the time
